backend/python/diffusers/backend.py

```diff
@@ -282,10 +282,17 @@ def LoadModel(self, request, context):
             if request.LoraAdapter:
                 # Check if its a local file and not a directory ( we load lora differently for a safetensor file )
                 if os.path.exists(request.LoraAdapter) and not os.path.isdir(request.LoraAdapter):
-                    self.load_lora_weights(request.LoraAdapter, 1, device, torchType)
+                    # self.load_lora_weights(request.LoraAdapter, 1, device, torchType)
+                    self.pipe.load_lora_weights(request.LoraAdapter)
```

Comment:
mmmh weird we didn't cough this up before

Reply:
ah well, right, that was not in the library before :) 

https://github.com/huggingface/diffusers/issues/3064#issuecomment-1702253404